openapi/ga4gh-tool-discovery.yaml

```diff
@@ -365,8 +365,11 @@ paths:
         Get a list of objects that contain the relative path and file type. The
         descriptors are intended for use with the
         /tools/{id}/versions/{version_id}/{type}/descriptor/{relative_path}
-        endpoint.
+        endpoint. If format=zip is specified, a zip file of all files available will be gotten instead.
```

Comment:
This is technically fine, but I think `gotten` has a bad reputation in certain circles 
https://grammarist.com/usage/got-gotten/

Please reword.